Scripts~/windows/.export-plugin-zip.ps1

```diff
@@ -0,0 +1,28 @@
+param(
+	[Parameter(Mandatory)]
+	[ValidatePattern("\d+\.\d+\.\d+")]
+	[string] $LightWeightUnrealServerSdkVersion,
+	[Parameter(Mandatory)]
+	[ValidatePattern("\d+\.\d+\.\d+")]
+	[string] $PluginVersion
+)
+
+$ROOT_DIR=Resolve-Path "$PSScriptRoot\..\.."
+$BUILD_DIR="$ROOT_DIR\GameLiftPlugin\build"
+$TEMP_PATH="C:\Temp\AmazonGameLiftPluginUnreal"
+$TEMP_PLUGIN_PATH="$TEMP_PATH\amazon-gamelift-plugin-unreal"
+$PACKAGED_ZIP_PATH="$BUILD_DIR\amazon-gamelift-plugin-unreal-$PluginVersion-sdk-$LightWeightUnrealServerSdkVersion.zip"
+
+New-Item -Force -Type Directory -Path $BUILD_DIR -ErrorAction Stop
```

Comment:
minor: move this after the Copy-Item step as this is leaving an empty "build" folder in the resulting release